Automattic/wp-calypso
# FSE NUX: fix the dimensions of the preview image

## Changes proposed in this Pull Request

To boldly fix the dimensions of the preview image to match the others where many have gone before.

## Testing instructions

Create a new account and site at `/new` and let yourself be swept away by the journey to the editor.

When the NUX modal appears, check that the images are of the same dimension and the jump affect is reduced.

Reported in https://github.com/Automattic/wp-calypso/pull/41966#pullrequestreview-409114042
